addons/mail/static/src/components/follower/follower.xml

```diff
@@ -5,7 +5,7 @@
         <div class="o_Follower">
             <t t-if="follower">
                 <a class="o_Follower_details" t-att-class="{ 'o-inactive': !follower.isActive }" href="#" t-on-click="_onClickDetails">
-                    <img class="o_Follower_avatar" t-attf-src="/web/image/{{ follower.resModel }}/{{ follower.resId }}/image_128" alt="Avatar"/>
+                    <img class="o_Follower_avatar" t-attf-src="/web/image/res.partner/{{ follower.partnerId }}/image_128" alt="Avatar"/>
```

Comment:
As per previous comment, this is URL is the value of `follower.partner.avatarUrl`, no need to keep it hardcoded in several places.